common/common.go

```diff
@@ -59,3 +59,55 @@ func GetDefaultMicroProfileLivenessProbe(ba BaseComponent) *corev1.Probe {
 func GetComponentNameLabel(ba BaseComponent) string {
 	return ba.GetGroupName() + "/name"
 }
+
+func CustomizeProbeDefaults(config *corev1.Probe, defaultProbe *corev1.Probe) *corev1.Probe {
+	probe := defaultProbe
+	if config.ProbeHandler.Exec != nil {
+		probe.ProbeHandler.Exec = config.ProbeHandler.Exec
+	}
+	if config.ProbeHandler.GRPC != nil {
+		probe.ProbeHandler.GRPC = config.ProbeHandler.GRPC
+	}
+	if config.ProbeHandler.HTTPGet != nil {
+		if probe.ProbeHandler.HTTPGet == nil {
+			probe.ProbeHandler.HTTPGet = &corev1.HTTPGetAction{}
+		}
+		if config.ProbeHandler.HTTPGet.Port.Type != 0 {
```

Comment:
Just wanted to confirm that a nil check on `config.ProbeHandler.HTTPGet.Port` is not needed here.